lms/djangoapps/courseware/views/views.py

```diff
@@ -2127,21 +2109,20 @@ def financial_assistance_form(request, course_id=None):
     if course_id:
         disabled = True
     enrolled_courses = get_financial_aid_courses(user, course_id)
-    incomes = ['Less than $5,000', '$5,000 - $10,000', '$10,000 - $15,000', '$15,000 - $20,000', '$20,000 - $25,000',
-               '$25,000 - $40,000', '$40,000 - $55,000', '$55,000 - $70,000', '$70,000 - $85,000',
-               '$85,000 - $100,000', 'More than $100,000']
 
-    annual_incomes = [
-        {'name': _(income), 'value': income} for income in incomes  # lint-amnesty, pylint: disable=translation-of-non-string
-    ]
+    default_course = None
+    for enrolled_course in enrolled_courses:
+        if enrolled_course['value'] == course_id:
+            default_course = enrolled_course['name']
+
```

Comment:
if course_id is unique here, i think it might make sense to add a `break` after line 2116, because there would be no need to iterate further through the list

Reply:
[curious] also looks like default_course previous was an empty string. why do we default it to None now instead of `''`

Reply:
It can work both ways in this case, as `form_field` template checks for the truthiness of this value